src/theory/bags/inference_generator.h

```diff
@@ -46,6 +46,12 @@ class InferenceGenerator
    *   (>= (bag.count e A) 0)
    */
   InferInfo nonNegativeCount(Node n, Node e);
+  /**
+   * @param n a node of integer type
```

Comment:
Is `n` a cardinality term?

Reply:
Yes, but this function is called on a skolem variable equals to a card term

Reply:
add this documentation
 ` @param n a node of integer type that equals to a card term`